src/mito2/src/manifest/storage.rs

```diff
@@ -380,7 +392,7 @@ impl ManifestObjectStore {
         let checkpoint_metadata = CheckpointMetadata {
             size: bytes.len(),
             version,
-            checksum: None,
+            checksum: Some(format!("{}", checksum)),
```

Comment:
nitpick: I believe that we could use `to_string()`.
```suggestion
            checksum: Some(checksum.to_string()),
```